release-managers.md

```diff
@@ -35,11 +35,29 @@ The responsibilities of each role are described below.
 
 ## Release Managers
```

Comment:
Should we add a note here that patch release team / branch manager used to be separate responsibilities but they are now under the umbrella of a release manager? It could be a bit confusing to see the different "team" documents and assume that not all release managers are part of those teams.

Reply:
@hasheddan That’s a good point, I agree that it could be a bit confusing. I’ll add a note.

Reply:
@hasheddan I've added a note.